### PR TITLE
Send booking request feature done

### PR DESCRIPTION
**Pull request by Mosiami87**
Created the send- booking request - work fine on the movies show page and saves/creates a booking request. 

This does not take care of the status - which we should define as 0, i.e. pending  and have workflow for changing this

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,10 +1,21 @@
 class BookingsController < ApplicationController
   def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+    @booking.movie = Movie.find(params[:movie_id])
+    @booking.save
+    redirect_to movies_path
```
Comment (dewaldreynecke) [inline app/controllers/bookings_controller.rb:7]: rather redirect to movie_path(@booking.movie)

File: app/views/movies/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,16 @@
-<h1>Movies#show</h1>
+
+<div class="container">
+  <h1><%= @movie.title %></h1>
+    <div class="card">
+      <h2> Create a booking</h2>
+      <%= simple_form_for [@movie, @booking] do |f| %>
+        <%= f.input :start_date %>
```
Comment (dewaldreynecke) [inline app/views/movies/show.html.erb:7]: Can we please change these dates to a date picker?

Comment (Mosiami87) [reply app/views/movies/show.html.erb:7]: Flatpickr added for choosing date

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (Mosiami87) [reply app/controllers/bookings_controller.rb:7]: redirect to movie_path(@booking.movie) done

**Pull request merged**